src/views/domain-workflows/hooks/use-list-workflows.ts

```diff
@@ -0,0 +1,57 @@
+'use client';
+import { useMemo } from 'react';
+
+import { useInfiniteQuery } from '@tanstack/react-query';
+import queryString from 'query-string';
+
+import {
+  type RouteParams as ListWorkflowsParams,
+  type ListWorkflowsResponse,
+  type ListWorkflowsRequestQueryParams,
+} from '@/route-handlers/list-workflows/list-workflows.types';
+import request from '@/utils/request';
+import { type RequestError } from '@/utils/request/request-error';
+
+import DOMAIN_WORKFLOWS_PAGE_SIZE from '../config/domain-workflows-page-size.config';
+
+export default function useListWorkflows({
+  params,
+  requestQueryParams,
```

Comment:
It is confusing from this hook context was is params vs requestQueryParams. 
Can we pass them from the parent as requestQueryParams ?

Reply:
Good idea